### DISCUSSION TITLE
Help with  access of Guest VMs directly without the need to create Nat or take console. (I am using Advanced zone without security groups)

### DISCUSSION THREAD

**Discussion by jack99trade** [apache/cloudstack#9539] [Q&A] [answered]
H Everyone, 
I am trying to achieve access of some Guest VMs directly from my user laptop.  I have done the Nat and it works but is there a way to assign public IP directly inside Guest VMs? 
 I have been trying to explore Shared networks with a dedicated vlan under cloudbr1 for guest traffic but no luck till now. 
Can someone guide me please in right direction.
Thank you

**Comment by pcfriek1987** [comment]
Not sure what you mean with "directly"? I don't think other then console is going to work for direct access as if you were attached to it.

**Comment by rajujith** [comment]
@jack99trade  Your approach to using the shared guest network is the right way to allocate public IP addresses to guest instance NICs. Why it didn't work for you?

**Reply by jack99trade** [reply, marked as answer]
> @jack99trade Your approach to using the shared guest network is the right way to allocate public IP addresses to guest instance NICs. Why it didn't work for you?

Hi @rajujith  it is working now.  I clicked Promiscuous mode while creating Shared network and now it works.

**Reply by weizhouapache** [reply]
Promiscuous mode is only available for vmware. do you use  vmware ? @jack99trade

**Reply by jack99trade** [reply]
> Promiscuous mode is only available for vmware. do you use vmware ? @jack99trade

@weizhouapache  yes base platform is ESXI :-  
![image](https://github.com/user-attachments/assets/fabcda99-5895-49b7-b7cf-9f91521ff6d3)

**Comment by DaanHoogland** [comment]
closed this for lack of interaction, please reopen or open a new item if applicable
